DataFormats/FTLRecHit/interface/FTLRecHit.h

```diff
@@ -0,0 +1,109 @@
+#ifndef DATAFORMATS_FTLRECHIT_H
+#define DATAFORMATS_FTLRECHIT_H 1
+
+#include "DataFormats/CaloRecHit/interface/CaloRecHit.h"
+#include <vector>
+
+/** \class FTLRecHit
+ *  
+ * based on EcalRecHit
+ *
+ * \author Lindsey Gray
+ */
+
+class FTLRecHit : public CaloRecHit {
```

Comment:
BTW, even EcalRecHit is no longer deriving from CaloRecHit (since ~71X).
Do you need this inheritance in FTL?